masu/celery/__init__.py

```diff
@@ -15,17 +15,30 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 """Celery module."""
+import logging
 
 from celery import Celery
 from celery.schedules import crontab
+from celery.signals import after_setup_logger
 
 from masu.config import Config
+from masu.util import setup_cloudwatch_logging
 
 
 # pylint: disable=invalid-name, redefined-outer-name
 celery = Celery(__name__, broker=Config.CELERY_BROKER_URL)
 
 
+@after_setup_logger.connect
+def setup_loggers(logger, *args, **kwargs):  # pylint: disable=unused-argument
```

Comment:
Please try to reformulate this to not require disabling pylint or please include a comment detailing why it was necessary to disable pylint.

Reply:
ok. I'll add a comment about how we don't need to use args, kwargs at this time.

Reply:
The code doesn't look like it uses them. Can they be removed?